web/client/examples/viewer/components/FullScreen.jsx

```diff
@@ -0,0 +1,61 @@
+var React = require('react');
+var {Button, Glyphicon} = require('react-bootstrap');
+
+var full = React.createClass({
+    propTypes: {
+        fullscreen: React.PropTypes.bool,
+        glyphicon: React.PropTypes.string,
+        buttonText: React.PropTypes.string,
+        propertiesToggle: React.PropTypes.func
+    },
+    componentWillReceiveProps: function(nextProps) {
+        if (nextProps.fullscreen !== this.props.fullscreen) {
```

Comment:
This is not correct: if the property changes, we need to check the new value to decide if entering or exiting fullscreen